packages/renderer/src/lib/container/ContainerDetailsLogs.svelte

```diff
@@ -33,17 +36,32 @@ let terminalParentDiv: HTMLDivElement;
 
 let logsTerminal: Terminal;
 
+let logsLength = 0;
+let skip = 0;
+let cancellableTokenId: number | undefined = undefined;
+let containerLogsClearedUnsubscriber: Unsubscriber | undefined = undefined;
+
+let containerDetailsLogsClearSetRevert: ((revert: boolean) => void) | undefined = undefined;
+
+$: {
```

Comment:
can't we just migrate to svelte5 instead of introducing svelte4 reactivity pattern?